src/data/map.js

```diff
@@ -458,12 +467,27 @@ async function getRaids(filter) {
                     (utils.inArray(filter.city, city) || filter.city.toLowerCase() === 'all')) {
                     const mapLink = util.format(config.google.maps, row.lat, row.lon);
                     raids.push({
-                        pokemon: `<img src='${imgUrl}' width=auto height=32 />&nbsp;${name}`,
-                        raid_starts: startTime,
-                        raid_ends: endTime,
-                        raid_level: 'Level ' + level,
+                        pokemon: {
+                            formatted: `<img src='${imgUrl}' width=auto height=32 />&nbsp;${name}`,
+                            sort: row.raid_pokemon_id
+                        },
+                        raid_starts: {
+                            formatted: startTime,
+                            sort: starts
+                        },
+                        raid_ends: {
+                            formatted: endTime,
+                            sort: ends
+                        },
+                        raid_level: {
+                            formatted: parseInt(level) === 6 ? 'Mega' : 'Level ' + level,
+                            sort: level
+                        },
                         gym_name: `<a href='${mapLink}' target='_blank'>${gym}</a>`,
-                        team: teamIcon,
+                        team: {
+                            formatted: teamIcon,
+                            sort: row.team_id
+                        },
```

Comment:
This seems unnecessary but ok.

Reply:
Oh I'm sure someone will complain that teams aren't sorted by 0, 1, 2, 3 so I think it is.